lib/controller/ProfielController.class.php

```diff
@@ -202,7 +202,7 @@ public function stamboom($uid = null) {
 
 	public function verjaardagen() {
 		require_once 'view/VerjaardagenView.class.php';
-		$body = new VerjaardagenView('alleverjaardagen');
+		$body = new AlleVerjaardagenView(null);

```

Comment:
Afschoppen van db calls is een controller taak, db calls doen is een model taak. Dit ga ik zo nog even fixen.